### DISCUSSION TITLE
Servo Type?

### DISCUSSION THREAD

**Discussion by artwhaley** [almtzr/Pedro#4] [Q&A]
When the instructions call for 360 degree micro servos -  are those continuous rotation servos, or are they normal back and forth servos with 360 degrees of total travel, and centered at 180?

**Comment by almtzr** [comment]
These are indeed continuous rotation servomotors, paired with a planetary gear system to provide better torque. It will also be possible to use standard 180° servomotors without the planetary gear system.
